fairscale/experimental/wgit/weigit_api.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import os
+import pathlib
+from pathlib import Path
+import shutil
+import sys
+from typing import Union
+
+import pygit2
+
+from fairscale.experimental.wgit.pygit import PyGit
+from fairscale.experimental.wgit.sha1_store import SHA1_store
+from fairscale.experimental.wgit.utils import ExitCode, create_dir, create_file, get_sha1_hash, write_to_file
```

Comment:
all these can be:

from .pygit import ...
from .sha1_store import ...
from .utils import ...